packages/react/src/components/button/Button.tsx

```diff
@@ -114,25 +155,26 @@ export const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
     return (
       <button
         ref={ref}
-        disabled={disabled}
+        disabled={disabled || isLoading}
         aria-disabled={isLoading || disabled || undefined}
         aria-label={isLoading ? loadingText : undefined}
         type="button"
         className={classNames(
           styles.button,
           styles[variant],
-          styles[`theme-${theme}`],
+          typeof theme === 'string' ? styles[`theme-${theme}`] : '',
           styles[`size-${size}`],
           fullWidth ? styles.fullWidth : '',
           isLoading ? styles.isLoading : '',
+          customThemeClass,
           className,
         )}
         onClick={isLoading ? loadingOnClick : onClick}
         {...rest}
       >
-        {isLoading ? <LoadingSpinner small /> : iconElementLeft}
-        <span className={styles.label}>{isLoading ? loadingText : children}</span>
-        {isLoading ? null : iconElementRight}
+        {isLoading ? <LoadingSpinner small /> : iconElementStart}
+        <span>{isLoading ? loadingText : children}</span>
+        {isLoading ? null : iconElementEnd}
```

Comment:
Side note: Opened a discussion about these old props in Slack.

Reply:
This I do not know? Let's discuss in daily perhaps